src/commands/helpcard/helpcard.command.ts

```diff
@@ -11,15 +9,26 @@ import { prefixedCommandRuleTemplate } from '../../config';
 })
 
 export class HelpcardCommand implements CommandClass {
+  action(msg: Message, args: string[]) {
+
+    // Return early if no argument is provided
+    if (args.length <= 1) {
+      msg.channel.send('Invalid command usage! You must specify a helpcard name: ``!helpcard "example_name_here"`` (without the quotes).');
+      return;
+    }
+
+    // Slice away the command and replace whitespace with underscores
+    let cardName = msg.content.slice(args[0].length).trim();
```

Comment:
Gotcha, in that case spaces make perfect sense! Did this work when testing though? Since args are split by the space character, `args[0]` should never have spaces in it right? I think you might need to join all the args by space and slice entry 0 for this to work how ya want.

Reply:
..I had it backwards: slice index 0, then join by space is what I meant.